packages/pasteup/palette.ts

```diff
@@ -25,74 +25,103 @@ export interface OtherColours {
         97: colour;
         100: colour;
     };
+    state: { success: colour; error: colour };
     specialReport: { dark: colour };
     labs: { dark: colour; main: colour };
     green: { dark: colour; main: colour };
     brand: { dark: colour; main: colour; pastel: colour };
 }
 
+const news: PillarColours = {
+    dark: '#ab0613',
+    main: '#c70000',
+    bright: '#ff4e36',
+    pastel: '#ffbac8',
+    faded: '#fff4f2',
+};
+
+const opinion: PillarColours = {
+    dark: '#bd5318',
+    main: '#e05e00',
+    bright: '#ff7f0f',
+    pastel: '#f9b376',
+    faded: '#fef9f5',
+};
+
+const sport: PillarColours = {
+    dark: '#005689',
+    main: '#0084c6',
+    bright: '#00b2ff',
+    pastel: '#90dcff',
+    faded: '#f1f8fc',
+};
+
+const culture: PillarColours = {
+    dark: '#6b5840',
+    main: '#a1845c',
+    bright: '#eacca0',
+    pastel: '#e7d4b9',
+    faded: '#fbf6ef',
+};
+
+const lifestyle: PillarColours = {
+    dark: '#7d0068',
+    main: '#bb3b80',
+    bright: '#ffabdb',
+    pastel: '#fec8d3',
+    faded: '#feeef7',
+};
+
+const green = {
+    dark: '#185E36',
+    main: '#22874D',
+};
+
+const brand = {
+    dark: '#041f4a',
+    main: '#052962',
+    pastel: '#506991',
+};
+
+const state = {
+    success: green.main,
```

Comment:
Are we ok that any change to `green.main` and `news.main` will be applied to `state.success` and `state.error`? If for example `news.main` all of a sudden becomes blue (😱) all error states would be blue, unless of course you knew to change the `news.main` === `state.error` mapping. Could we not just use the code for these `state` properties?

Reply:
ah! good point. my rationale here was the same fear you have but the other way around – what if we tweak the news red or the greens just a bit (same colour but say more contrasty or something) and these fall off the grid? Wd normally hardcode them twice as you suggest so I'm happy to do that!

Reply:
perhaps we could do them the other way around? semantic keys first, and the pillars inherit from them

Reply:
Why not:

```js
red: { main: '#XXXXXX'}
news: { main: red.main }
state: { error: red.main }
```

Reply:
done!

Reply:
I'm happy with this solution 👍